dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -304,7 +304,7 @@ public static Receiver internalReprOf(
                 }
                 break;
             case MEMBER_SELECT:
-                receiver = internalRepOfMemberSelect(provider, (MemberSelectTree) receiverTree);
+                receiver = internalReprOfMemberSelect(provider, (MemberSelectTree) receiverTree);
```

Comment:
There are other methods in this class that are named `internalRepOf` instead of the more consistent `internalReprOf`.
Those methods were public so I didn't want to change them. It would be good to make this more uniform, either as part of this PR or separately.